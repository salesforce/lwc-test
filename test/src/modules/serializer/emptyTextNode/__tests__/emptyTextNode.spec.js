/*
 * Copyright (c) 2018, salesforce.com, inc.
 * All rights reserved.
 * SPDX-License-Identifier: MIT
 * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
 */
import { createElement } from 'lwc';
import EmptyTextNode from '../emptyTextNode';

it('serializes component with empty text nodes', () => {
    const elm = createElement('serializer-component', { is: EmptyTextNode });
    document.body.appendChild(elm);

    expect(elm).toMatchSnapshot();
});
