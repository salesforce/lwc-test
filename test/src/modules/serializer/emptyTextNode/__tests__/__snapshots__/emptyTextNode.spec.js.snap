// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`serializes component with empty text nodes 1`] = `
<serializer-component>
  <div>
    <h1>
      hello
    </h1>
  </div>
</serializer-component>
`;
